aws/templates/common.ftl

```diff
@@ -1151,9 +1162,23 @@ behaviour.
     [/#if]
 
     [#return
-        attributeIfContent(
+        attributeIfTrue(
```

Comment:
could have stuck with attributeIfContent and dropped the ?has_content checks because attributeIfContent does that anyway.

Reply:
Can I check two attributes for content though?